config/ActionEventsConfig.xml

```diff
@@ -16,6 +16,7 @@
 			isDisabledCallbackFunc: is used to disable a action event after a action event refresh or on load.
 			callbackState: isDisabledCallbackFunc parameter, this is an int!
 			text: is the action event text, by default it returns courseplay:loc(name), where 'name' is the previous defined action event name.
+			textVisibilityNeeded: Is disabling action event text forbidden ? (default: false) 
```

Comment:
very confusing description (maybe it is due to my age :) but disabling forbidden is false is just too much boolean logic for me...